src/topology/algebra/group.lean

```diff
@@ -67,10 +67,18 @@ by { ext, refl }
 lemma is_open_map_mul_left (a : G) : is_open_map (λ x, a * x) :=
 (homeomorph.mul_left a).is_open_map
 
+lemma left_coset_open {U : set G} (x : G) (h : is_open U) :
+is_open (left_coset x U) :=
+is_open_map_mul_left x _ h
```

Comment:
If you do it like this, then we can use dot notation:
```suggestion
lemma is_open.left_coset {U : set G} (h : is_open U) (x : G) : is_open (left_coset x U) :=
is_open_map_mul_left x _ h
```

Also, we should also add the additive version of this. It would look like this:
```lean
lemma is_open.left_add_coset
  {G : Type*} [topological_space G] [add_group G] [has_continuous_add G]
  {U : set G} (h : is_open U) (x : G) : is_open (left_add_coset x U) :=
is_open_map_add_left x _ h
```
except you should not write it manually like this and should use `to_additive`. For some reason `to_additive` doesn't know to transport `left_coset` to `left_add_coset`.

I suggest asking for help about this on Zulip.